lib/honeycomb/client.rb

```diff
@@ -22,6 +22,10 @@ def initialize(configuration:)
       @client.add_field "meta.beeline_version", Honeycomb::Beeline::VERSION
       @client.add_field "meta.local_hostname", configuration.host_name
 
+      integrations = Honeycomb.integrations_to_load
+      @client.add_field "meta.instrumentations_count", integrations.count
+      @client.add_field "meta.instrumentations", integrations.map(&:to_s).to_s
```

Comment:
it would be good to add these fields to https://github.com/honeycombio/beeline-ruby/blob/master/spec/support/event_data_shared_examples.rb#L3 so that they are included in the base tests